src/Main.ls

```diff
@@ -54,6 +57,7 @@ package
         override public function run():void
         {
             TextField.registerBitmapFont( BitmapFont.load( "assets/arialComplete.fnt" ), "SourceSansPro" );
+            DeviceCapabilities.dpi = 300;
```

Comment:
I did to poor effect. Screenshot incoming.

On Fri, Feb 12, 2016 at 6:54 PM, Miha Lunar <notifications@github.com>
wrote:

> In src/Main.ls
> <https://github.com/TheEngineCompany/HackForACause/pull/2#discussion_r52821802>
> :
>
> > @@ -54,6 +57,7 @@ package
> >          override public function run():void
> >          {
> >              TextField.registerBitmapFont( BitmapFont.load( "assets/arialComplete.fnt" ), "SourceSansPro" );
> > +            DeviceCapabilities.dpi = 300;
>
> Try DeviceCapabilities.dpi = Platform.getDPI();
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TheEngineCompany/HackForACause/pull/2/files#r52821802>
> .
>